src/navigation/sectionalTabs/sectionalTabs.jsx

```diff
@@ -424,7 +423,7 @@ class SectionalTabs extends Component {
         return reducedItems;
     }
 
-    getTabProps(tab) {
+    getTabProps(tab = {}) {
```

Comment:
Nice catch, I was getting an error in the client side :P thank you very much @morethanfire